src/core/Scene.ts

```diff
@@ -685,6 +694,8 @@ export class Scene extends SharedAPI {
 	override disconnectedCallback() {
 		super.disconnectedCallback()
 		this.#stopParentSizeObservation()
+		this.removeEventListener('click', this.#handleClick)
+		this.removeEventListener('pointermove', this.#handlePointermove)
```

Comment:
- [ ] we don't need to explicitly remove, as event an element with its own event listeners will be GC'd fine.